### ISSUE TITLE
./install.sh

### ISSUE THREAD

**Issue by dcrand2012** [silvanmelchior/RPi_Cam_Web_Interface#413]
When I run ./install.sh or sudo ./install.sh get this message:
Reading package lists... Done
Building dependency tree
Reading state information... Done
Package dialog is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

**Comment by roberttidey** [comment]
I did an install check about a week ago and it was OK then.

Did you do a sudo apt-get update first? I believe some of the repository locations have been changed and one does need to update the indexes.

**Comment by dcrand2012** [comment]
Just did sudo apt-get update again for the third time and now it installed
Thanks

**Comment by roberttidey** [comment]
Yes. Sometimes the raspberry repositories can have some temporary glitches.
